dev-app/build.gradle

```
@@ -43,6 +43,7 @@ composeCompiler {
 
 dependencies {
     implementation project(path: ':sdk')
+    implementation project(path: ':prebid')
```

```suggestion
    implementation project(":sdk")
    implementation project(":prebid")
```

in the other build.gradle, the path is not there.